source/whats-new.txt

```diff
@@ -35,6 +36,22 @@ Learn what's new in:
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+
+.. _version-5.1:
+
+What's New in 5.1
+-----------------
+
+.. _java-new-features-5.1:
+
+New Features in 5.1
+~~~~~~~~~~~~~~~~~~~
+
+The 5.1 driver release includes the following new features:
+
+- Support for building native applications by using the GraalVM native-image tool. These
```

Comment:
We have begun testing with GraalVM native image technology, building and running a native application using the GraalVM native-image tool in our CI. However, we have not introduced specific support for end-users in `5.1`; I don't think it should be considered as a feature. Creating a native-image had been possible before 5.1, we only started actively testing this capability in version 5.1.